src/config.ts

```diff
@@ -96,15 +96,24 @@ function verifyTypeMatches(
  * Configuration (from file)
  */
 export class Configuration {
+  private _initialDevelopment = true;
+
   allowedBranches = ".*";
-  initialDevelopment = true;
   maxSubjectLength = 80;
   releaseBranches = /^release\/.*\d+\.\d+\.*$/;
   versionScheme = "semver";
   prereleasePrefix?: string = undefined;
   tags: IConfigurationRules = DEFAULT_ACCEPTED_TAGS;
   rules: Map<string, IRuleConfigItem> = new Map<string, IRuleConfigItem>();
 
+  set initialDevelopment(initialDevelopment: boolean) {
+    this._initialDevelopment = initialDevelopment;
+  }
+
+  get initialDevelopment(): boolean {
+    return this._initialDevelopment;
+  }
```

Comment:
...why? This adds nothing and just breaks the spread operator.

Reply:
This allows me to spy on the `getter` 😢